.github/workflows/release-tests.yml

```diff
@@ -0,0 +1,101 @@
+name: Release Testing
```

Comment:
So this will trigger all of these jobs in parallel and since all of them use the same aws/gcp account it will cause issues. So for now let's use `depends_on` so that for each cloud we only run one of them at a time. But eventually let's use different environment names etc.